### PR TITLE
Feature/g0 5341 Show additional columns (TCS, TDS, Tax, Discount) in sales and purchase register

### PR DESCRIPTION
**Pull request by prateek-giddh**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.html [hunk 1]
```diff
@@ -302,36 +302,36 @@
             </table>
         </div>
 
-        <div class="text-center " *ngIf="(PurchaseRegisteDetailedResponse$ | async).totalItems > 50">
+        <div class="text-center " *ngIf="(purchaseRegisteDetailedResponse$ | async).totalItems > 50">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.html:305]: Please change 50 by pagination constant

File: apps/web-giddh/src/app/reports/components/salesRegister-expand-component/sales.register.expand.component.html [hunk 2]
```diff
@@ -302,36 +302,36 @@
             </table>
         </div>
 
-        <div class="text-center " *ngIf="(SalesRegisteDetailedResponse$ | async).totalItems > 50">
+        <div class="text-center " *ngIf="(salesRegisteDetailedResponse$ | async).totalItems > 50">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/salesRegister-expand-component/sales.register.expand.component.html:305]: Please replace 50 by pagination constant

File: apps/web-giddh/src/app/reports/components/salesRegister-expand-component/sales.register.expand.component.ts [hunk 3]
```diff
@@ -61,7 +61,7 @@ export class SalesRegisterExpandComponent implements OnInit {
     bsValue = new Date();
 
     constructor(private store: Store<AppState>, private invoiceReceiptActions: InvoiceReceiptActions, private activeRoute: ActivatedRoute, private router: Router, private _cd: ChangeDetectorRef, private _generalActions: GeneralActions) {
-        this.SalesRegisteDetailedResponse$ = this.store.pipe(select(p => p.receipt.SalesRegisteDetailedResponse), takeUntil(this.destroyed$));
+        this.salesRegisteDetailedResponse$ = this.store.pipe(select(p => p.receipt.SalesRegisteDetailedResponse), takeUntil(this.destroyed$));
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/reports/components/salesRegister-expand-component/sales.register.expand.component.ts:64]: Rename 'p'

File: apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.ts [hunk 4]
```diff
@@ -54,7 +54,7 @@ export class PurchaseRegisterExpandComponent implements OnInit {
     bsValue = new Date();
 
     constructor(private store: Store<AppState>, private invoiceReceiptActions: InvoiceReceiptActions, private activeRoute: ActivatedRoute, private router: Router, private _cd: ChangeDetectorRef, private _generalActions: GeneralActions) {
-        this.PurchaseRegisteDetailedResponse$ = this.store.pipe(select(p => p.receipt.PurchaseRegisteDetailedResponse), takeUntil(this.destroyed$));
+        this.purchaseRegisteDetailedResponse$ = this.store.pipe(select(p => p.receipt.PurchaseRegisteDetailedResponse), takeUntil(this.destroyed$));
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.ts:57]: Rename 'p'

**Pull request merged**